accepted/2020/mono-convergence/security-support-new-platforms.md

```diff
@@ -0,0 +1,90 @@
+# Security support for new .NET 5 platforms
+
+## Overview
+Currently in .NET Core, we have a solid [strategy](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/cross-platform-cryptography.md) for dealing with cryptography on Linux and OSX.  At the native level, `System.Security.Cryptography.Native` relies upon OpenSSL for Linux and `System.Security.Cryptography.Native.Apple` relies upon platform specific API’s for OSX.  
```

Comment:
> https://github.com/dotnet/corefx/blob/master/Documentation/architecture/cross-platform-cryptography.md

This link is broken. May be you are referring to [this link](https://github.com/dotnet/runtime/blob/master/docs/design/features/cross-platform-cryptography.md) instead.